wgpu-core/src/binding_model.rs

```diff
@@ -477,21 +475,15 @@ pub struct BindGroupLayout<A: HalApi> {
     #[allow(unused)]
     pub(crate) binding_count_validator: BindingTypeMaxCountValidator,
     pub(crate) info: ResourceInfo<BindGroupLayout<A>>,
-    pub(crate) label: String,
 }
 
 impl<A: HalApi> Drop for BindGroupLayout<A> {
     fn drop(&mut self) {
+        resource_log!("Drop {}", self.error_ident());
```

Comment:
nit: The `resource_log` being in the branch was meant to have logging when something interesting happens (the bind group layout actually gets destroyed). If we hoist it out of the branch, then we won't be able to tell whether the bind group was snatched.

Reply:
Done!

Reply:
Thanks!